root/insurance.py

```diff
@@ -6,18 +6,28 @@
 
 class Client:
     def __init__(self):
-        self.baseURL = "https://sandbox.root.co.za/v1/insurance"
-        self.appID = os.environ.get('ROOT_APP_ID')
-        self.appSecret = os.environ.get('ROOT_APP_SECRET')
+        self.sandBox = os.environ.get('ROOT_SANDBOX', True)
+        self.production = False if self.sandBox else True
+        self.prodUrl = "https://api.root.co.za/v1/insurance"
+        self.baseURL = "https://sandbox.root.co.za/v1/insurance" if self.sandBox else self.prodUrl
+        self.appSecret = os.environ.get('ROOT_APP_SECRET') if not self.sandBox else ""
```

Comment:
Not sure if this PR is the place but is it not better to have these default to `None` if not provided so that an exception can bubble up later down the line since it's impossible to access the Root API without a secret key (as far as I am aware; it also seems like the right pattern)?

Reply:
Can do, as far as I know the auth differs on prod, was just a foolproof idea I had.